src/Parser.js

```diff
@@ -967,15 +967,15 @@ export default class Parser {
         if (symbols[this.mode][text]) {
             if (this.settings.strict && this.mode === 'math' &&
                 extraLatin.indexOf(text) >= 0) {
-                this.settings.nonstrict("unicodeTextInMathMode",
+                this.settings.reportNonstrict("unicodeTextInMathMode",
                     `Latin-1/Unicode text character "${text[0]}" used in ` +
                     `math mode`, nucleus);
             }
             symbol = new ParseNode(symbols[this.mode][text].group,
                             text, this.mode, nucleus);
         } else if (supportedCodepoint(text.charCodeAt(0))) {
             if (this.settings.strict && this.mode === 'math') {
```

Comment:
Not part of this PR, but `this.settings.strict` check seems not necessary. (same for above)

Reply:
It's not necessary; it is an optimization, so the string didn't have to be generated unless strict mode was set to something that needed it (see the comment for `reportNonstrict`). This was probably more useful when the default for `strict` was `false`, though, making this a common case.